### PR TITLE
pointing at an item in your hands shows it off

### PR DESCRIPTION
**Pull request by AndroBetel**
# About the pull request

just like that "Show held item" verb

# Explain why it's good for the game

it's hard for me personally to find that verb in object tab sooooo

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

yep...

</details>

# Changelog
:cl:
qol: Pointing at an item in your hands shows it off.
/:cl:

### CODE REVIEW

**Comment by ihatethisengine** [comment]
there is a keybind for it lmao but good pr

**Comment by AndroBetel** [comment]
>there is a keybind for it lmao but good pr

i only have so much keys on my keyboard

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1687,3 +1687,11 @@
...
 	INVOKE_ASYNC(target, TYPE_PROC_REF(/mob/living/carbon/human, update_body), 1, 0)
 	INVOKE_ASYNC(target, TYPE_PROC_REF(/mob/living/carbon/human, update_hair))
+
+/mob/living/carbon/human/point_to_atom(atom/A, turf/T)
+	if(isitem(A))
+		var/obj/item/item = A
+		if(item == get_active_hand() || item == get_inactive_hand())
+			item.showoff(src)
+			return TRUE
+	..()
```
Comment (fira) [inline code/modules/mob/living/carbon/human/human.dm:1697]:
```suggestion
	return ..()
```

**Review by fira — CHANGES_REQUESTED** [review]

**Comment by HumiliatedGoblin** [comment]
would it be possible to make this for equipped things, like armor, belts and backpacks too?

**Comment by ihatethisengine** [comment]
> would it be possible to make this for equipped things, like armor, belts and backpacks too?

To use "show held item" verb on items you don't hold?

**Review by Drulikar — APPROVED** [review]

**Pull request merged**